src/libraries/System.Memory/src/System/Buffers/SequenceReader.Search.cs

```diff
@@ -680,6 +680,22 @@ public long AdvancePastAny(T value0, T value1)
             return Consumed - start;
         }
 
+        /// <summary>
+        /// Position the reader at the end of the sequence
```

Comment:
nit:
```suggestion
        /// Moves the reader to the end of the sequence.
```

Looking at existing API docs, looks like we either use "Advances" or "Moves". Maybe we should pick one of those rather than "Positions":
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.sequencereader-1?view=netcore-3.0

Reply:
yep, sorry Ahson didn't take a deeper look at docs guide, I'll try to keep into account in future.